src/rust/ide/src/view/node_editor.rs

```diff
@@ -699,26 +710,52 @@ impl GraphEditorIntegratedWithControllerModel {
         Ok(())
     }
 
+    fn expression_entered_in_ui
+    (&self, local_call:&Option<LocalCall>) -> FallibleResult<()> {
+        if let Some(local_call) = local_call {
+            let method_pointer = local_call.definition.clone();
+            let expression_id  = local_call.call;
+            let controller     = self.controller.clone_ref();
+            let logger         = self.logger.clone_ref();
+            let graph_editor   = self.editor.clone_ref();
+            let enter_action   = async move {
+                match controller.enter_method_pointer(expression_id,&method_pointer).await {
+                    Ok(_) => {
+                        info!(logger,"Entering node.");
```

Comment:
The purpose if this log is to log that we start entering node. It shouldnt be emitted after it is done. (that's why it isn't in the past tense)